README.md

```diff
@@ -0,0 +1,109 @@
+# HDF5 VOL Tests
+
+This repository contains several test applications that exercise [HDF5](https://github.com/HDFGroup/hdf5)'s
+public API and serve as regression tests for HDF5 [VOL Connectors](https://portal.hdfgroup.org/display/HDF5/Virtual+Object+Layer).
+
+### Software Requirements
+
+The following are required to build the HDF5 VOL tests:
+
+- CMake 2.8.12.2 or newer
+- HDF5 1.12.0 or newer
+
+Note: The HDF5 VOL tests project is currently unsupported for building on Windows.
+
+### Build Process
+
+The HDF5 VOL tests are built using CMake in an out-of-source fashion. A typical build
+would look like:
+
+    mkdir build
+    cd build
+    cmake -DCMAKE_BUILD_TYPE:STRING=Release ${BUILD_OPTIONS} -G ${CMAKE_GENERATOR} ..
+    make
+
+Build time options that can be passed for `${BUILD_OPTIONS}` are discussed in the following
+section. The CMake documentation should be consulted when determining what to set for
+`CMAKE_GENERATOR`.
+
+### Build Options
+
+The following build options are available to influence how the VOL tests get built. To set an
+option, it should be prepended with `-D` when passed to the `cmake` command. For example,
+
+    cmake -DHDF5_DIR=/temp/hdf5 ..
+
+would set the value of the `HDF5_DIR` build option to `/temp/hdf5`.
+
+##### Required
+
+`HDF5_DIR` - The VOL test build process will attempt to automatically locate an HDF5 installation
+on the system. However, if one cannot be found, then this option should be set to point to a
+directory which contains the CMake-related files for an HDF5 installation. For example, on linux
+systems this directory will usually be:
+
+    <hdf5_install_root>/share/cmake/hdf5/
+
+and will contain `hdf5-config.cmake`, among other files.
+
+If the HDF5 installation pointed to was built with the parallel library enabled, the `CMAKE_C_COMPILER`
+build option may also need to be set to point to an MPI compiler wrapper so that CMake can find the MPI libraries necessary for building, e.g.:
+
+    cmake -DCMAKE_C_COMPILER=/temp/openmpi/bin/mpicc ..
+
```

Comment:
You should not use the wrappers as the compilers. CMake should find the MPI install by itself using find_package.

Reply:
Sure, that works fine if MPI is in a standard location. Otherwise, the FindMPI CMake module says to set this. Maybe I should mention this is only for non-standard installs?